payroll/src/sass/styles.scss

```
@@ -695,3 +695,166 @@ body {
   pointer-events: none;
   opacity: 0.5;
 }
+
+/* Scoped styles from Register */
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700;800;900&display=swap');
+  .register-register-wrapper { min-height:100vh; display:flex; font-family:'Inter',sans-serif; background:#f0f4f8; }
```

Stylelint (17.11.1)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)